cypress/integration/2-settings/stake-abpt.ts

```diff
@@ -0,0 +1,26 @@
+import { configEnvWithTenderlyMainnetFork } from '../../support/steps/configuration.steps';
+
+describe('Checking ABPT token modal on Staking page', () => {
+  describe('CASE1:Open Stake page', () => {
+    configEnvWithTenderlyMainnetFork({});
+
+    it('step1:Open Stake page', () => {
+      cy.get('a[href*="/staking/"]').click();
```

Comment:
Could target cypress attribute here, that way test wouldn't break if we changed the link name
`cy.get('[data-cy="menuStake"]')`